pkg/core/native/native_gas.go

```diff
@@ -79,15 +80,21 @@ func (g *GAS) Initialize(ic *interop.Context) error {
 
 // OnPersist implements Contract interface.
 func (g *GAS) OnPersist(ic *interop.Context) error {
-	//for _ ,tx := range ic.block.Transactions {
-	//	g.burn(ic, tx.Sender, tx.SystemFee + tx.NetworkFee)
-	//}
-	//validators := g.NEO.getNextBlockValidators(ic)
-	//var netFee util.Fixed8
-	//for _, tx := range ic.block.Transactions {
-	//	netFee += tx.NetworkFee
-	//}
-	//g.mint(ic, <primary>, netFee)
+	for _, tx := range ic.Block.Transactions {
+		g.burn(ic, tx.Sender, big.NewInt(tx.SystemFee.IntegralValue()+tx.NetworkFee.IntegralValue()))
+	}
+	array := g.NEO.getNextBlockValidators(ic, nil)
```

Comment:
`GetValidatorsInternal`, we want to give some GAS for *current* primary validator.